src/auth/oauth/test/oauth.test.ts

```diff
@@ -266,7 +267,8 @@ describe('validateAuthCallback', () => {
     expect(session?.accessToken).toBe(successResponse.access_token);
   });
 
-  test('requests access token for valid callbacks with online access and updates session with expiration and onlineAccessInfo', async () => {
+  test('requests access token for valid callbacks with online access and updates session' +
+    ' with expiration and onlineAccessInfo', async () => {
```

Comment:
I don't love breaking the string here, but we can always rewrite it something more sensibly long if we ever refactor this test.